src/server/config/env/runTime.js

```diff
@@ -237,6 +237,21 @@ const runTime = [
       }
     },
   },
+  {
+    name: 'ONE_ENABLE_SERVER_TRACING',
+    defaultValue: 'false',
+    normalize: (input) => {
+      if (input.toLowerCase() === 'true') {
```

Comment:
looks like we have duplicate code for this, it would be nice to abstract it into a function to reuse the code, something like `...normalizeAndValidateBoolean()`

Reply:
This function was removed anyway